src/get-data.ts

```diff
@@ -40,15 +49,34 @@ const getData = async (): Promise<PRData> => {
   if (info.repository && info.sender) {
     prData.actor = info.sender.login;
   } else {
-    throw new Error('Issue with context');
+    throw new Error('Error: Issue with context');
   }
 
-  const coverageData = getInput('coverageData');
-  const commentData = await getCoverage(coverageData);
+  let coverageData;
+
+  try {
+    coverageData = getInput('coverageData');
+  } catch (error) {
+    throwError(error, 'Failed to grab file ./coverage/lcov.info.');
```

Comment:
An error here doesn't mean the file doesn't exist it means the `coverageData` argument wasn't set. We also define a default value for this so this should never happen.